cli/dts/lib.deno.unstable.d.ts

```diff
@@ -7,15 +7,10 @@ declare namespace Deno {
   export interface BenchDefinition {
     fn: () => void | Promise<void>;
     name: string;
+    group?: string;
     ignore?: boolean;
-    /** Specify number of iterations benchmark should perform. Defaults to 1000. */
-    n?: number;
-    /** Specify number of warmup iterations benchmark should perform. Defaults
-     * to 1000.
-     *
-     * These iterations are not measured. It allows the code to be optimized
-     * by JIT compiler before measuring its performance. */
-    warmup?: number;
+    /** Benchmark should be used as the baseline for other benchmarks */
+    baseline?: boolean;
```

Comment:
What happens if multiple cases use `baseline: true`? Can you add it to the docstring?